Update README.md (#10493)
elastic/integrations: packages/security_detection_engine/docs/README.md

# Prebuilt Security Detection Rules

The detection rules package stores the [prebuilt security rules](https://www.elastic.co/guide/en/security/current/prebuilt-rules.html) for the Elastic Security [detection engine](https://www.elastic.co/guide/en/security/current/detection-engine-overview.html).

> This package is automatically added for the security users and does not require user actions.

Reviewer: ```suggestion
This package is automatically added for Elastic Security users and doesn't require any user action. 
```
